app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @recipe.name %></h1>
+  <p> Complexity(1-10): <%= @recipe.complexity %> </p>
+  <p> <%= @recipe.genre %> </p>
+  <p> <% @recipe_ingredients.each do |ingredient| %> </p>
```

Comment:
Like I said above, since you set this instance variable to be **all ingredients**, and not the ones that belong to this recipe, this is actually going through every single ingredient in the table instead of the ones that belong to this specific recipe, which doesn't meet the requirement of User Story 2.